app/src/main/java/com/codepath/apps/critterfinder/fragments/SwipeablePetsFragment.java

```diff
@@ -0,0 +1,63 @@
+package com.codepath.apps.critterfinder.fragments;
+
+import android.os.Bundle;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.codepath.apps.critterfinder.R;
+import com.squareup.picasso.Picasso;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+/**
+ * Our tinder-like pet browser
+ */
+public class SwipeablePetsFragment extends Fragment {
+
+    @Bind(R.id.text_pet_gender) TextView mPetGender;
+    @Bind(R.id.text_pet_name) TextView mPetName;
+    @Bind(R.id.image_pet) ImageView mPetImage;
```

Comment:
most of this is just temporary to show some fake data in our swipeable pets fragment